specifyweb/frontend/js_src/lib/components/Interactions/PrepDialog.tsx

```diff
@@ -133,7 +136,10 @@ export function PrepDialog({
           </>
         )
       }
-      header={interactionsText.preparations()}
+      header=
+          {interactionsText.preparations({
+            preparationTable: tables.Preparation.label,
+        })}
```

Comment:
interesting that prettier did not reformat this file in https://github.com/specify/specify7/pull/6110/commits/0371f027fa8349965873396f247440e92828b18d

part of https://github.com/specify/specify7/issues/6051 I guess